pyinfra/modules/virtualenv.py

```diff
@@ -35,7 +35,9 @@ def virtualenv(
 
     elif present and not host.fact.directory(path):
         # Create missing virtualenv
-        command = '/usr/bin/virtualenv -p {}'.format(python)
+        command = '/usr/bin/virtualenv'
```

Comment:
Could we change this to just `virtualenv`? If Python is installed to `/usr/local` pip will install virtualenv to `/usr/local/bin/virtualenv` - I think it's best to just execute the command and have the system locate it.